podcast/the-changelog-300.md

```diff
@@ -297,10 +297,7 @@ I feel like in a lot of readmes, a lot of open source website, it'll say right t
 **Adam Stacoviak:** We were pretty close to that one though. I think there were a lot of community members knee deep into the Io and Node fork and bringing the communities back together. From my perspective, I think there were plenty of community members that were what I would consider just peer-to-peer developers, not corporations that were -- I guess they were leading Io, but then they were also bringing it back to this recombination of Io/Node.
```

Comment:
Adam was saying you could add `.js` after `Io` to make it clear that he was referring to `IO.js`, but that's not required by any means to get this merged.